### ISSUE TITLE
Specs are failing due to possible changes how capybara matches text (after gem update)

### ISSUE THREAD

**Issue by yivo** [openware/peatio#49]
I have this spec failing randomly.

```
  1) Sign in display captcha after too many failed attempts
     Failure/Error: expect(page).not_to have_content(I18n.t('simple_form.labels.session.captcha'))
       expected not to find text "Captcha" in "Trade Solvency Sign In Sign Up Login Login my account Login Peatio now for you to exchange. Forgot Password? Sign Up × Wrong customer ID or password,please try again. * Sign In E-mail * Sign In Password Set a password which has at least 6 characters and it's better to contain uppercase and lowercase letters and numbers. * Captcha Refresh page to get new captcha. Cancel Maintained by the Helios team with the help of our contributors. Code licensed under MIT, documentation under CC BY 3.0. Currently v0.2.2GitHubDocumentsAboutIssuesReleases". (However, it was found 2 times using a case insensitive search.)
     # ./spec/features/sign_in_spec.rb:27:in `block (2 levels) in <top (required)>'
```

**Comment by yivo** [comment]
@mod Assigned to Valera

**Comment by yivo** [comment]
To reproduce: `bundle exec rspec --seed 21424`

**Comment by yivo** [comment]
Fixed in #59
